src/nft/components/explore/WelcomeModal.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import Modal from 'components/Modal'
+import { useState } from 'react'
+import { X } from 'react-feather'
+import styled, { useTheme } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const Container = styled.div`
+  position: relative;
+  display: flex;
+  padding: 30% 24px 24px;
+  overflow: hidden;
+  height: fit-content;
+  user-select: none;
+`
+
+const CloseButton = styled(X)`
+  position: absolute;
+  top: 20px;
+  right: 24px;
+  cursor: pointer;
+`
+
+const Link = styled.a`
+  outline: none;
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+`
```

Comment:
General question: is there a better way to handle links within the app? This looks like something you'd typically want to do for every link!

Reply:
Yes pls look at StyledInternalLink or StyledLink from src/theme/components/index.tsx